source/GlassView.Export/GlassView.Export.csproj

```diff
@@ -10,8 +10,13 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Atmoos.World.IO" Version="0.0.2" />
     <PackageReference Include="BenchmarkDotNet" Version="0.13.12" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Using Remove="System.IO" />
```

Comment:
I assume this removes the reference for "System.IO" from the global usings. Is this enough to ensure that no one can reference System.IO in a future class or was the intention just to remove it from the assemblies?